content/home/examples/a-component-using-external-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Component Using External Plugins
+title: Un componente usando Plugins externos
 order: 3
 domid: markdown-example
 ---
 
-React is flexible and provides hooks that allow you to interface with other libraries and frameworks. This example uses **remarkable**, an external Markdown library, to convert the `<textarea>`'s value in real time.
+React es flexible y proporciona medios que te permiten interactuar con otras bibliotecas y frameworks. Este ejemplo usa **remarkable**, una biblioteca externa de Markdown, que convierte los valores de `<textarea>` en tiempo real.
```

Comment:
What do you think about `librerías` instead of `bibliotecas`?

Reply:
I'd like to take into account @carburo's contribution [here](https://github.com/reactjs/es.reactjs.org/pull/3/commits/d36a07a732046050b39169872e3ffbb6a744e079#r252787383).